### ISSUE TITLE
Fix versionning for releases/tags

### ISSUE THREAD

**Issue by landryb** [mapproxy/mapproxy#307]
## Context

The version field in setup.py is used to show the version on the demo pages, along with the date. It is also used when defining the name of the .egg-info dir when packaging (ie i patch it to use MapProxy-${GH_TAGNAME}-py${MODPY_VERSION}.egg-info/)

## Expected Behavior

The version value should match the tagged/released version

## Actual Behavior

it always sports a 'a' (for alpha?) and sometimes doesn't match the tag.

https://github.com/mapproxy/mapproxy/blob/1.10.0/setup.py#L57 -> 1.10.0a
https://github.com/mapproxy/mapproxy/blob/1.9.1/setup.py#L57 -> 1.8.2a
https://github.com/mapproxy/mapproxy/blob/1.9.0/setup.py#L57 -> 1.8.2a
https://github.com/mapproxy/mapproxy/blob/1.8.0/setup.py#L57 -> 1.8.0a

## Possible Fix

Fix the version tag as part of the release process when pushing the tag ? We cant do anything for the previous releases, but can something be done for the future ones ?

In OpenBSD, i fix it this way: http://cvsweb.openbsd.org/cgi-bin/cvsweb/ports/geo/mapproxy/Makefile?rev=1.8&content-type=text/x-cvsweb-markup

**Comment by olt** [comment]
The official release packages on https://pypi.python.org/pypi/MapProxy do have the right version. But yes, there is no commit with version='1.10.0'. I only set the final release version when I build the packages and then bump the version to 1.11.0a, etc. to avoid unofficial releases by accident. 

I made a change to the release script. This makes sure that there is a commit with the right version for all future releases.
